### PR TITLE
Create parity_accounts namespace and implement newAccountFromSecret

### PR DESCRIPTION
**Pull request by CPerezz**
All has been tested on a crate I've made (also with the rpc tests) and it looks good. Here I attach some evidences:

Happy to improve or change whatever is necessary!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ For more see [examples folder](./examples).
 
 ### Parity-specific APIs
 - [ ] Parity read-only: `parity_*`
-- [ ] Parity accounts: `parity_*`
+- [x] Parity accounts: `parity_*` (Currently being implemented @CPerezz)
```
Comment (tomusdrw) [inline README.md]: Let's change this to mention that it's only partially implemented instead. Feel free to add yourself to `CONTRIBUTORS.md` file or sth like this.

File: src/api/mod.rs [hunk 2]
```diff
@@ -7,6 +7,7 @@ mod net;
 mod personal;
 mod traces;
 mod web3;
+mod parity_accounts;
```
Comment (tomusdrw) [inline src/api/mod.rs]: Can you move this up to keep the list sorted?

File: src/api/mod.rs [hunk 3]
```diff
@@ -15,6 +16,8 @@ pub use self::net::Net;
 pub use self::personal::Personal;
 pub use self::web3::Web3 as Web3Api;
 pub use self::traces::Traces;
+pub use self::parity_accounts::ParityAccounts;
```
Comment (tomusdrw) [inline src/api/mod.rs]: same as above

File: src/api/parity_accounts.rs [hunk 4]
```diff
@@ -0,0 +1,58 @@
+use api::Namespace;
+use helpers::{self, CallFuture};
+use types::{Address, H256};
+use Transport;
+
+
```
Comment (tomusdrw) [inline src/api/parity_accounts.rs]: too many blank lines - one should be fine.

File: src/api/parity_accounts.rs [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        ParityAccounts { transport }
+    }
+
+    fn transport(&self) -> &T {
+        &self.transport
+    }
+}
+
+
```
Comment (tomusdrw) [inline src/api/parity_accounts.rs]: same as above and below

File: src/api/parity_accounts.rs [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+impl<T: Transport> ParityAccounts<T> {
+    /// Imports account from Sk.
```
Comment (tomusdrw) [inline src/api/parity_accounts.rs]:
```suggestion
    /// Imports account given secret and password.
```

**Comment by CPerezz** [comment]
All the proposals have been corrected. Hope I didn't miss something. 

Notify me if there's something more that needs to be refactored.

**Comment by tomusdrw** [comment]
Thanks!

**Pull request merged**